Makefile

```diff
@@ -80,7 +80,7 @@ crds.clean:
 	@find package/crds -name '*.yaml.sed' -delete || $(FAIL)
 	@$(OK) cleaned generated CRDs
 
-terrajet.run:
+terrajet.run: $(GOIMPORTS)
```

Comment:
i don't see this being needed in provider-tf-aws, so i'd be suspicious of needing to add it 🤔 

https://github.com/crossplane-contrib/provider-tf-aws/blob/main/Makefile

Reply:
The provider-tf-aws project has a different cmd/generator/main.go from the template project.

I added this as an attempt to pass earlier GH runs that failed.

Reply:
I've got `check-diff` getting further with this change https://github.com/crossplane-contrib/provider-tf-equinix-metal/pull/1/commits/6d897c40a72a4307fc873b87a3b0187ade40afde, but now the `make generate` (`make reviewable`) and `make check-diff`  have different expectations.